web/cypress/e2e/auto-tax-filings.spec.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import {
+  clickModalSaveButton,
+  completeBusinessStructureTask,
+  openFormationDateModal,
+  selectDate,
+  selectLocation,
+} from "@businessnjgovnavigator/cypress/support/helpers/helpers";
+import { completeNewBusinessOnboarding } from "@businessnjgovnavigator/cypress/support/helpers/helpers-onboarding";
+import { LookupIndustryById } from "@businessnjgovnavigator/shared/";
+import { onDashboardPage } from "cypress/support/page_objects/dashboardPage";
+
+describe("auto tax filing [feature] [all] [group4]", () => {
+  beforeEach(() => {
+    cy.loginByCognitoApi();
+  });
+
+  it("successfully completes auto tax filing with business name and tax Id", () => {
+    const industry = LookupIndustryById("cosmetology");
+    const legalStructureId = "limited-liability-company";
+    const businessName = "Make Me Pretty";
+
+    completeNewBusinessOnboarding({ industry });
+
+    completeBusinessStructureTask({ legalStructureId });
+
+    onDashboardPage.getEditProfileLink().should("exist");
+    cy.get('[data-task="apply-for-shop-license"]').first().click();
+    cy.get('button[data-testid="cta-secondary"]').first().click();
+    cy.intercept(`${Cypress.env("API_BASE_URL")}/api/users/*`).as("userAPI");
+
+    cy.visit("/profile");
+    cy.get('input[data-testid="businessName"]').type(businessName);
+    cy.get('button[data-testid="save"]').first().click();
+    openFormationDateModal();
+    selectDate("04/2021");
+    selectLocation("Allendale");
+    clickModalSaveButton();
+
+    cy.get('[data-testid="register-for-taxes"]').first().click();
+    cy.get('input[name="taxId"]').type("123456789098");
+    cy.get("button").contains("Save").click();
+    cy.get(`[data-testid="back-to-dashboard"]`).first().click({ force: true });
+    cy.get('[data-testid="cta-funding-nudge"]').first().click();
+    cy.get('[data-testid="get-tax-access"]').should("not.exist");
+    cy.get('[data-testid="alert-content-container"]').should("exist");
+  });
```

Comment:
Should we add something in here to check that one of the tax filing events is listed in the calendar?